apps/coordinator/ts/proof/__tests__/proof.service.test.ts

```diff
@@ -120,23 +93,6 @@ describe("ProofGeneratorService", () => {
     await expect(service.generate(defaultProofArgs)).rejects.toThrow(ErrorCodes.PRIVATE_KEY_MISMATCH.toString());
   });
 
-  test("should throw error if poll is not found", async () => {
```

Comment:
Why was this test removed?

Reply:
I created this issue to follow up this comment #2639